File: Modul-05/SOAL 1/PRAK501-2310817210009-MuhammadIbnuSina.py
def MAXBILANGAN(A, B, C, D):
    return max(A, B, C, D)

if __name__ == "__main__":
    A, B, C, D = map(int, input().split())
    hasil = MAXBILANGAN(A, B, C, D)
    print(hasil)

File: Modul-05/SOAL 1/PRAK501-2310817210009-MuhammadIbnuSina.c
#include <stdio.h>

int MAXBILANGAN(int A, int B, int C, int D) {
    int MAX = A;
    if (B > MAX) {
        MAX = B;
    }
    if (C > MAX) {
        MAX = C;
    }
    if (D > MAX) {
        MAX = D;
    }
    return MAX;
}

int main() {
    int A, B, C, D;
    scanf("%d %d %d %d", &A, &B, &C, &D);
    int HASIL = MAXBILANGAN(A, B, C, D);
    printf("%d", HASIL);
    return 0;
}
